### ISSUE TITLE
[feature request] add option to NOT let Android launcher know about the new wallpaper colors

### ISSUE THREAD

**Issue by Bobbydew1** [cvzi/darkmodewallpaper#203]
[bug] The options to let the android launcher know the new wallpaper colors are making my launcher glitch/crash when i use different wallpaper for my lock screen from my home screen

**Comment by cvzi** [comment]
Have you tried to disable the "Immediately inform Android about new wallpaper colors after unlock" option in the advanced settings?

**Comment by Bobbydew1** [comment]
> Have you tried to disable the "Immediately inform Android about new wallpaper colors after unlock" option in the advanced settings?

Yes i did but its the same, it does work in a few seconds but then it starts glitching lagging my launcher when it informs the colors of the wallpaper 

So its like this
I turn on the **imediately** option 
Then when i unlock my phone the launcher glitches and crash 

Then if i turn on the **wait**  
It doesn't glitches/crash for a few seconds but after that it will crash my launcher
It also slows my luancher too

**Comment by cvzi** [comment]
OK I'll add such an option

**Comment by Bobbydew1** [comment]
> OK I'll add such an option

Instant 5 star rating 🙏

**Comment by cvzi** [comment]
I have added the option in the advanced settings. 

At the moment it only prevents the lock screen colors. The home screen colors are still sent after unlocking (but they are the same as before the lock screen so the launcher shouldn't have to change anything) .
If this is not enough I can change it to not send the colors at all or only if the mode changes from light to dark.
